src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4708,7 +4705,7 @@ public override bool PreVisit(CallNode node)
                 var overloads = LookupFunctions(funcNamespace, node.Head.Name.Value);
                 if (!overloads.Any())
                 {
-                    if (funcNamespace.ToString() != string.Empty)
+                    if (!string.IsNullOrEmpty(funcNamespace.ToString()))
```

Comment:
>IsNullOrEmpty [](http://example.com/codeflow?start=32&length=13)

this change the behavior on null.

Reply:
well, not really
if funcNamespace is null, it was throwning
if .ToString() would return null, it would if ToString would return null
In that case ToString maps to ToDottedSyntax() which never returns null.
We're 100% safe here.

That said, having code that differentiates null and an empty string is a real bad practice or you need a super good reason.